<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color:rgb(62,52,64);
	border:2px solid rbg(255,255,255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="file_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border:1px solid rgb(255,255,255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="audio_duration">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="play_pause">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border:none;
  border-radius:40px;
  background-color:grey;
  color:white;
  min-width:80px;
  max-width:80px;
  min-height:80px;
  max-height:80px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoRepeatInterval">
     <number>104</number>
    </property>
   </widget>
   <widget class="QPushButton" name="audio_forward">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border:none;
  border-radius:40px;
  background-color:grey;
  color:white;
  min-width:80px;
  max-width:80px;
  min-height:80px;
  max-height:80px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoRepeatInterval">
     <number>104</number>
    </property>
   </widget>
   <widget class="QPushButton" name="audio_backward">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border:none;
  border-radius:40px;
  background-color:grey;
  color:white;
  min-width:80px;
  max-width:80px;
  min-height:80px;
  max-height:80px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoRepeatInterval">
     <number>104</number>
    </property>
   </widget>
   <widget class="QPushButton" name="mute_unmute">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	


}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="audio_volume">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>68</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalTime">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>68</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
